schemas/upgrade.json

```diff
@@ -22,6 +22,14 @@
             "sectionBegin": "Upgrade",
             "propertyOrder": 10
         },
+        "vsd_pre_upgrade_check_script": {
```

Comment:
For Schema changes, we need to run the generate job to regenerate examples.

Reply:
completed generating examples